DatadogRUM/Sources/RUMMonitor/Scopes/RUMViewScope.swift

```diff
@@ -552,6 +552,10 @@ internal class RUMViewScope: RUMScope, RUMContextProvider {
 
         var commandAttributes = command.attributes
         let errorFingerprint = commandAttributes.removeValue(forKey: RUM.Attributes.errorFingerprint) as? String
+        var timeSinceAppStart: Int64? = nil
+        if let startTime = context.launchTime?.launchDate {
```

Comment:
I feel it's still valid to report it even if the app has been pre-warmed. 
It could serve as a useful indication to the user that this error occurred after the app has been pre-warmed, which might otherwise not be the case no cold launches. 

It also aligns with Android's behavior.